src/test.js

```diff
@@ -242,9 +242,11 @@ Test.prototype = {
 				processHooks( test, module.parentModule );
 			}
 
-			if ( module.hooks[ handler ].length ) {
-				for ( let i = 0; i < module.hooks[ handler ].length; i++ ) {
-					hooks.push( test.queueHook( module.hooks[ handler ][ i ], handler, module ) );
+			const hooksForHandler = module.hooks[ handler ];
+
+			if ( hooksForHandler && hooksForHandler.length ) {
```

Comment:
This will solve the current issue, but ideally, these hooks should never be getting invoked when `hooksForHandler` is not defined. I'd like to see if we can find a fix for that (which I would consider the root issue here).

Reply:
Ya I agree. This is a bit of a stopgap, but was the most visible issue that was addressable. Agree that further investigation is warranted.